javascripts/italian.js

```diff
@@ -1 +1,16 @@
-"use strict"
\ No newline at end of file
+"use strict"
+
+var translate = (function(oldFunc) {
+  var italian = {
+    happy: "Buono",
+    christmas: "Natale"
+  };
+
+  return {
+      translateToItalian: function(language) {
+      console.log("test");
+      return italian[language];
+    }
+  // return oldFunc;
+  };
+})(oldFunc || {});
```

Comment:
Need to pass in translate here so that function continues from previous js files.